testing/endpoint_multihop.go

```diff
@@ -217,6 +219,25 @@ func (endpoint *EndpointM) AcknowledgePacket(packet channeltypes.Packet, initPro
 	return endpoint.Chain.sendMsgs(ackMsg)
 }
 
+// TimeoutPacket sends a MsgTimeout to the channel associated with the endpoint.
+func (endpoint *EndpointM) TimeoutPacket(packet channeltypes.Packet, initProofHeight exported.Height) error {
```

Comment:
hah, the linter is complaining about using "endpoint" instead of "ep" here. let's make the linter happy.

Reply:
also applies to AcknowledgePacket above.